src/components/Settings/AccountTab.react.js

```diff
@@ -444,16 +492,26 @@ AccountTab.propTypes = {
   accessToken: PropTypes.string,
   actions: PropTypes.object,
   avatarType: PropTypes.string,
+  userEmailId: PropTypes.string,
 };
 
 function mapStateToProps(store) {
+  const userSettingsViewedByAdmin = store.settings.userSettingsViewedByAdmin;
+  const {
+    email,
+    userName,
+    timeZone,
+    prefLanguage,
+    avatarType,
+  } = userSettingsViewedByAdmin;
```

Comment:
Can we write, something like.

`const settings = email !== '' ? store.settings.userSettingsViewedByAdmin : store.settings;`

Then userName can be `settings.userName`